### PR TITLE
BTS-54: fix an assertion

### PR DESCRIPTION
**Pull request by graetzer**
### Scope & Purpose

fix a shutdown issue

- [x] Bug-Fix for *devel-branch* (i.e. no need for backports?)
- [x] The behavior change can be verified via automatic tests

### Testing & Verification

This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

File: arangod/Aql/SharedQueryState.h [hunk 1]
```diff
@@ -17,7 +17,7 @@
 ///
 /// Copyright holder is ArangoDB GmbH, Cologne, Germany
 ///
-/// @author Jan Steemann
+/// @author Jan Steemannk
```
Comment (jsteemann) [inline arangod/Aql/SharedQueryState.h:20]:
```suggestion
/// @author Jan Steemann
```

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/9996/

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/9997/

**Comment by jsteemann** [comment]
Starting another run:
http://172.16.10.101:8080/job/arangodb-matrix-pr/10003/

**Comment by graetzer** [comment]
finally http://172.16.10.101:8080/job/arangodb-matrix-pr/10007/

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10032/

**Pull request merged**